crud/common/sharding_key.lua

```diff
@@ -0,0 +1,9 @@
+local sharding_metadata_module = require('crud.common.sharding.sharding_metadata')
+
+local sharding_key_cache = {}
+
+function sharding_key_cache.update_cache(space_name)
+    return sharding_metadata_module.update_sharding_key_cache(space_name)
+end
```

Comment:
I added a comment about this problem. For sharding key:

```diff
+ -- This method is exported here because
+ -- we already have customers using old API
+ -- for updating sharding key cache in their
+ -- projects like `require('crud.common.sharding_key').update_cache()`
function sharding_key_cache.update_cache(space_name)
    return sharding_metadata_module.update_sharding_key_cache(space_name)
end
```

For sharding func:

```diff
+ -- This method is exported here because
+ -- we already have customers using old API
+ -- for updating sharding key cache in their
+ -- projects like `require('crud.common.sharding_key').update_cache()`
+ -- This method provides similar behavior for
+ -- sharding function cache.
function sharding_func_cache.update_cache(space_name)
    return sharding_metadata_module.update_sharding_func_cache(space_name)
end
```

Reply:
Thanks!